backends/pkg/reconciler/eventmesh/handler.go

```diff
@@ -73,6 +89,33 @@ func BuildEventMesh(listers Listers, logger *zap.SugaredLogger) (EventMesh, erro
 		}
 	}
 
+	// build 2 maps for event types for easier access
+
+	// map key: "<namespace>/<eventType.spec.type>"
+	etTypeMap := make(map[string]*EventType)
```

Comment:
rename to `etByNamespacedType` ?

Reply:
I removed this map when I simplified the trigger processing